src/rgw/motr/synchronization/rgw_motr_gc_sync.h

```diff
@@ -0,0 +1,47 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * SAL implementation for the CORTX Motr backend
+ *
+ * Copyright (C) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ *
+ */
+#ifndef __MOTR_GC_SYNCHROZATION__
+#define __MOTR_GC_SYNCHROZATION__
+
+#include "../../rgw_sal_motr.h"
```

Comment:
no need to use `../../` the root directory is by default `src/rgw`

Reply:
done